README.md

```diff
@@ -9,47 +9,52 @@
 
 # Overview
 
-- [Installation and Usage](#-installation-and-Usage)
+- [Installation](#-installation)
+- [Usage](#-usage)
 - [Documentation](#-documentation)
 - [Support](#-support)
 - [FAQs](#-faqs)
 - [Contributing](#-contributing)
 
-## 🧰 Installation and Usage
+## 🧰 Installation
 
-**Install**
+The easiest way to install spectral is to use either [npm](https://www.npmjs.com/):
 
 ```bash
 npm install -g @stoplight/spectral-cli
+```
 
-# OR
+Or [yarn](https://yarnpkg.com/):
 
+```
 yarn global add @stoplight/spectral-cli
 ```
 
-Find more [installation methods](https://meta.stoplight.io/docs/spectral/ZG9jOjYyMDc0Mw-installation) in our documentation.
+You can find more installation methods [here](https://meta.stoplight.io/docs/spectral/ZG9jOjYyMDc0Mw-installation).
+
+## 💻 Usage
 
-**Create a local ruleset**
+### 1. Create a local ruleset
 
-Spectral, being a generic YAML/JSON linter, needs a ruleset to lint files. There are two ways to get a ruleset:
+Spectral, being a generic YAML/JSON linter, **needs a ruleset** to lint files. A ruleset is a JSON or YAML file (often the file will be called `.spectral.yaml`) that contains a collection of rules, which can be used to lint other JSON or YAML files such as an API description.
```

Comment:
Could also be JavaScript or TypeScript now. Our docs for this need work but you can see how I've done it over here. https://apisyouwonthate.com/blog/distribute-spectral-style-guides-with-npm#publish-your-spectral-style-guide-to-npm

Reply:
Mhm, what do you think is the best way to word this? Maybe we just add JavaScript/TypeScript:

"Spectral, being a generic YAML/JSON linter, **needs a ruleset** to lint files. A ruleset is a JSON, YAML, or JavaScript/TypeScript file (often the file will be called `.spectral.yaml` for a YAML ruleset) that contains a collection of rules, which can be used to lint other JSON or YAML files such as an API description."

Reply:
@heitortsergent that works.